specification/refarch.adoc

```diff
@@ -3,142 +3,184 @@
 [[refarch]]
 == Reference Architecture Details
 
-We describe the capabilities of the platform to support memory isolation 
-requirements for confidentiality of workloads in TVMs. We then describe 
-the properties of the TSM, its instantiation, isolation and operational model 
-for the TVM life cycle. The description in this section refers to the reference 
+We describe the capabilities of the platform to support memory isolation
+requirements for confidentiality of workloads in TVMs. We then describe
+the properties of the TSM, its instantiation, isolation and operational model
+for the TVM life cycle. The description in this section refers to the reference
 architecture in Figure 1.
 
```

Comment:
This change seems to be just removal of white space. Would be good to put such editorial commits in a separate single commit. Also having one change per commit helps review commit by commit.